orangecontrib/spectroscopy/widgets/owspectra.py

```diff
@@ -1009,6 +1067,71 @@ def grid_apply(self):
         self.plot.showGrid(self.show_grid, self.show_grid, alpha=0.3)
         self.show_grid_a.setChecked(self.show_grid)
 
+    def find_peak_variables(self):
+        if self.maximum_point is None:
+            self.maximum_point = np.amax(self.data.X[:, self.data_xsind])
+        if self.minimum_point is None:
+            self.minimum_point = np.amin(self.data.X[:, self.data_xsind])
+        if self.start_point is None:
+            self.start_point = round(np.median(getx(self.data)), 2)
+
+    def peak_apply(self, position):
+        if self.viewtype == INDIVIDUAL:
+            Label_line = VerticalPeakLine()
+            Label_line.setMovable(True)
+            Label_line.setPen(pg.mkPen(color=QColor(Qt.black), width=2, style=Qt.DotLine))
+            Label_line.setSpan(mn=self.minimum_point, mx=self.maximum_point)
```

Comment:
Why is this setSpan needed? Why not draw Infinite lines also out of the data set?